docs/pages/desktop-access/getting-started.mdx

```diff
@@ -314,6 +314,62 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
   ![Disable Require](../../img/desktop-access/disable.png)
 </Figure>
 
+### Configure certificate for RDP connections
```

Comment:
It looks like this is the final subsection of "Step 3/6: Configure a GPO to allow Teleport connections". I'm not sure that's the intention, though, since the second paragraph begins with "In this step we'll...". 

If this is the beginning of a new step, we'll need to promote this section heading to an H2 (using `##` instead of `###`), add "Step n/d:" to the title, and renumber the rest of the H2 section headings (e.g., this would be "Step 4/7").

Reply:
I was thinking about whether I should add it as a separate step or add it to the existing one as it's related to it. After I gave it some thought and re-read it again, I agree that it make more sense to make it a separate optional step.